### DISCUSSION TITLE
Stream to downcast with Logo (and text)

### DISCUSSION THREAD

**Discussion by msman0** [datarhei/restreamer#548] [Q&A] [answered]
Hi, how can I add the logo also in the stream to external services? Is it possible to add (dynamic) text from a file in the stream?
Thank you and best regards.

**Comment by svenerbeck** [comment, marked as answer]
Hello @msman0 
I'm sorry, but at the moment, it is not possible to add the logo to the stream for external services or to include dynamic text from a file in the stream.

Cheerio
Sven
